Makefile

```diff
@@ -90,19 +90,22 @@ deploy-kind: check-env-docker-repo git-commit-sha patch-kind manifests deploy-na
 	kustomize build config/crd | kubectl apply -f -
 	kustomize build config/default/overlays/kind | kubectl apply -f -
 
-generate-installation-manifests:
-	mkdir -p installation
-	kustomize build config/namespace/base/ > installation/namespace.yaml
-	kustomize build config/crd/ > installation/crd.yaml
-	kustomize build config/rbac/ > installation/rbac.yaml
-	kustomize build config/installation > installation/operator.yaml
-
 generate-helm-manifests:
 	kustomize build config/namespace/base/ > charts/operator/templates/namespace.yaml
 	kustomize build config/crd/ > charts/operator/templates/crd.yaml
 	kustomize build config/rbac/ > charts/operator/templates/rbac.yaml
 	kustomize build config/default/overlays/helm/ > charts/operator/templates/deployment.yaml
 
+install_dir = config/installation/generated
+generate-installation-manifest:
+	mkdir -p $(install_dir)
+	mkdir -p releases
+	cp config/namespace/base/namespace.yaml $(install_dir)/namespace.yaml
```

Comment:
Remove the cp if we can use sibling resources in kustomization.yml under config/installation